hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/spi/impl/TestOzoneManagerServiceProviderImpl.java

```diff
@@ -441,6 +478,17 @@ private OzoneManagerProtocol getMockOzoneManagerClient(
     return ozoneManagerProtocolMock;
   }
 
+  // Mock the case of SNNFE
+  private OzoneManagerProtocol getMockOzoneManagerClientWithThrow()
+      throws IOException {
+    OzoneManagerProtocol ozoneManagerProtocolMock =
+        mock(OzoneManagerProtocol.class);
+    when(ozoneManagerProtocolMock.getDBUpdates(any(OzoneManagerProtocolProtos
+        .DBUpdatesRequest.class)))
+        .thenThrow(new SequenceNumberNotFoundException());
```

Comment:
do we also need to check the SequenceNumberNotFoundException in unit test?

Reply:
Actually, the sync request is sent through client protocol, and recon would create an IOException instead of SequenceNumberNotFoundException when receiving the response from OM. 
Here is for a clear case statement, maybe the exception conversion could be implemented someday with specific **ResultCode**